openlibrary/plugins/upstream/addbook.py

```diff
@@ -1009,11 +1010,10 @@ def process_input(self, i):
         if 'author' in i:
             author = trim_doc(i.author)
             alternate_names = author.get('alternate_names', None) or ''
-            author.alternate_names = [
-                name.strip()
-                for name in alternate_names.replace("\n", ";").split(';')
-                if name.strip()
+            alternate_names = [
+                name.strip() for name in alternate_names.split('\n') if name.strip()
             ]
+            author.alternate_names = uniq(alternate_names, key=str.lower)
```

Comment:
This can be a little tighter here
```suggestion
            author.alternate_names = uniq([
                name.strip() for name in alternate_names.split('\n') if name.strip()
            ], key=str.lower)
```